style.scss

```diff
@@ -0,0 +1,631 @@
+}
+
+/*
+2. Remove default margin
+*/
+* {
+    margin: 0;
+}
+
+/*
+3. Allow percentage-based heights in the application
+*/
+html,
+body {
+    height: 100%;
+}
+
+/*
+6. Improve media defaults
+*/
+img,
+picture,
+video,
+canvas,
+svg {
+    display: block;
+    max-width: 100%;
+}
+
+/*
+7. Remove built-in form typography styles
+*/
+input,
+button,
+textarea,
+select {
+    font: inherit;
+}
+
+/*
+8. Avoid text overflows
+*/
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    overflow-wrap: break-word;
+}
+
+
+/* Remove list styles on ul, ol elements with a list role, which suggests default styling will be removed */
+ul[role='list'],
+ol[role='list'] {
+    list-style: none;
+    padding: 0;
+}
+
+/*
+  Typographic tweaks!
+  4. Add accessible line-height
+  5. Improve text rendering
+*/
+
+
+/* UTILITY CLASSES */
+
+.flex {
+    display: flex;
+    
+    &.justify-between {
+        justify-content: space-between;
+    }
+    
+    &.items-center {
+        align-items: center;
+    }
+}
+
+.text {
+    &--center {
+        text-align: center;
+    }
+}
+
+.button {
+    cursor: pointer;
+    font-weight: var(--fw-bold);
+    font-size: var(--fs-button);
+    line-height: 1;
+    color: var(--clr-neutral-100);
+    background-color: var(--clr-primary-900);
+    text-decoration: none;
+    border: 0;
+    border-radius: 100vmax;
+    padding: var(--size-400) var(--size-500);
+    max-height: 3.5rem;
+
+    &:hover,
+    &:focus-visible {
+        background-color: var(--clr-primary-900-hover);
+    }
+
+    &[data-type='alternative'] {
+        background-color: var(--clr-accent-400);
+
+        &:hover,
+        &:focus-visible {
+            background-color: var(--clr-accent-400-hover);
+        }
+    }
+    
+    &--previous,
+    &--next{
+        border-radius: 50%;
+    }
+}
+
+.card {
+    border-radius: var(--size-200);
+}
+
+.description{
+    font-size: var(--fs-body);
+    font-weight: var(--fw-regular);
+    line-height: var(--fs-550);
+    color: var(--clr-neutral-600);
+    
+    &--white{
+        color: var(--clr-neutral-100);
+    }
+}
+
+.title {
+    font-weight: var(--fw-bold);
+    &--small {
+        font-size: var(--fs-small-heading);
+        line-height: var(--fs-600);
+    }
+    
+    &--medium {
+        font-size: var(--fs-medium-heading);
+    }
+    
+    &--large {
+        font-size: var(--fs-large-heading);
+        line-height: var(--fs-900);
+    }
+}
+
+
+/* general styling */
+body {
+    -webkit-font-smoothing: antialiased;
+    font-family: var(--ff-body);
+    font-size: var(--fs-body);
+    font-weight: var(--fw-regular);
+    color: var(--clr-primary-900);
+}
+
+.container {
+    --max-width: 1110px;
+    --padding: 1rem;
+
+    width: min(var(--max-width), 100% - (var(--padding) * 2));
+    margin-inline: auto;
+}
+
+.header {
+    padding: var(--size-400) 0;
+}
+
+.portfolio__intro {
+    display: flex;
+    flex-direction: column;
+    padding-block: var(--size-900);
+    
+    .description {
```

Comment:
If it's utility - it shouldn't be nested under portfolio, if it's modifier, it should look like it.